voir/instruments/gpu/cpu.py

```diff
@@ -3,7 +3,7 @@ def is_installed():
 
 
 class DeviceSMI:
-    def get_gpus_info(self):
+    def get_gpus_info(self, selection=ModuleNotFoundError):
```

Comment:
What is `ModuleNotFoundError` doing here?

Reply:
probably some copy paste error, fixed